src/osdc/Objecter.cc

```diff
@@ -3435,6 +3435,7 @@ void Objecter::handle_osd_op_reply(MOSDOpReply *m)
     // new tid
     s->ops.erase(op->tid);
     op->tid = ++last_tid;
+    s->ops[op->tid] = op; // add op to ops again
```

Comment:
@jdurgin and @liewegas could you take a look also? if my analysis makes sense, we need to have this fix in luminous, and probably need to backport it.